lib/web_ui/lib/src/engine/alarm_clock.dart

```diff
@@ -33,7 +33,11 @@ class AlarmClock {
   DateTime? _datetime;
 
   /// The callback called when the alarm goes off.
-  late ui.VoidCallback callback;
+  ///
+  /// If this is null, the alarm goes off without calling the callback. Set the
+  /// callback to null of the callback is a closure holding onto expensive
```

Comment:
Typo: `/// callback to null -if- the callback is a closure holding onto expensive`

Reply:
Done.